README.md

```diff
@@ -1,26 +1,34 @@
 # Lento: A Modern Distraction Blocker
 
-Lento is a modern distraction blocker for Windows and macOS, built with Python. There are a lot of similar products out there, but we built Lento because we couldn't find an alternative with all of the features we wanted.
+Lento is a modern distraction blocker for Windows and macOS, built with
+Python. There are a lot of similar products out there, but we built Lento
+because we couldn't find an alternative with all of the features we wanted.
 
 ## Features
-- Set different goals for different things, each with their own blocklists and notification presets
-- Ability to block both apps and websites
-- Can easily set preset time lengths for different goals
-- Automatically disables your ability to turn a session off before the block time runs out
-- "Soft blocks" where you are given a prompt when opening an app/site instead of an outright block
-- Persistent block that survives across uninstalls, logouts, and reboots
-- Browser agnostic (works with your favourite browser, whether it's Chrome, Firefox, Edge, Safari, Vivaldi, or another!)
-- Advanced notification functionality: remind yourself of your goals at set intervals/on idle/when switching apps
-- Clean, easy to use UI
-- Works across Windows and macOS (Linux support coming soon)
+
+- Set different goals for different things, each with their own blocklists and
+notification presets.
+- Ability to block both apps and websites.
+- Can easily set preset time lengths for different goals.
+- Automatically disables your ability to turn a session off before the block
+time runs out.
+- "Soft blocks" where you are given a prompt when opening an app/site instead
+of an outright block.
+- Persistent block that survives across uninstalls, logouts, and reboots.
+- Browser agnostic (works with your favourite browser, whether it's Chrome,
+Firefox, Edge, Safari, Vivaldi, or another!).
+- Advanced notification functionality: remind yourself of your goals at set
+intervals/on idle/when switching apps.
+- Clean, easy to use UI.
+- Works across Windows and macOS (Linux support coming soon).
 - Free and open source!
 
 ## Installing
-You can download the latest release [here]() or from the [Releases page]() on the GitHub Repo.
-
-If you want to build from source:
 
-[TODO: Build from source instructions]
+Work is in progress on installers. For now, you'll need to clone the
+repository and run `app.py` directly.
```

Comment:
Here are the aforementioned changes considered in contention.

Feel free to resolve this if they're appropriate, or just ask and I'll revert them.